### PR TITLE
Use selenium=4.* find_element

### PR DESCRIPTION
**Pull request by stefanvodita**
As promised, here is another PR. [Selenium is at version 4 now and the API for finding elements has been updated](https://www.selenium.dev/documentation/webdriver/getting_started/upgrade_to_selenium_4/).
I had to make these changes to run the code.

At this point I've gotten an appointment too, so I can no longer validate new code. What do you think of putting together a test where the webdriver is mocked so development can continue?
If that sounds worthwhile to you, I'll see what I can come up with.

### CODE REVIEW

**Comment by Ed1123** [comment]
Now I remember I didn't continue with the development of the script. It was
because I got an appointment and I couldn't test it anymore. Did the script
help you with yours?

The test would be great to continue the development, but I'm not sure how
to implement it. I'd like to hear your ideas. ^^

**Comment by stefanvodita** [comment]
I pushed another commit with tests using local html files.
Let me know what you think @Ed1123

File: creds.py
Comment (Ed1123) [inline creds.py:9]: I suspect this change is because of a line break? Can you confirm please?

**Review by Ed1123 — COMMENTED** [review]
So far it looks great! Thanks for contributing. 
I would like to check if I can refactor the code a little bit more before merging it.
Also, I'm more familiar with pytest so I was considering maybe converting the tests to it.

File: creds.py
Comment (stefanvodita) [reply creds.py:9]: It's some trailing whitespace that got removed

**Comment by stefanvodita** [comment]
> Also, I'm more familiar with pytest so I was considering maybe converting the tests to it.

This works with pytest too. Run ` pytest test_selenium_scraper.py`.
Let me know if you want me to make any other changes.

**Comment by stefanvodita** [comment]
Hi @Ed1123  , this PR has been on hold for a while. Are you considering a different approach?

**Review by Ed1123 — APPROVED** [review]
Sorry for the delay. I wanted to check the code in detail before merging. I really appreciate your contributions. :)

**Pull request merged**
